code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -22,7 +21,9 @@
 	var/tmp/particle_move_delay       = 10                          ///lower is faster
 	var/tmp/sound_spray               = 'sound/effects/spray2.ogg'  ///Sound played when spraying
 	var/safety                        = FALSE                       ///Whether the safety is on
-	
+
+/obj/item/chems/spray/solvent_can_melt(var/solvent_power = MAT_SOLVENT_STRONG)
+	return FALSE // maybe reconsider this
```

Comment:
Probably not necessary unless we make the contents inside chem objects able to melt them.